plugins/csi/plugin.go

```diff
@@ -101,6 +105,11 @@ type CSIPlugin interface {
 	// for the given volume.
 	NodeUnpublishVolume(ctx context.Context, volumeID, targetPath string, opts ...grpc.CallOption) error
 
+	// NodeExpandVolume is used to expand a volume. This MUST be called after
+	// any ControllerExpandVolume is called, but only if that RPC indicates
+	// that node expansion is required
+	NodeExpandVolume(ctx context.Context, req *NodeExpandVolumeRequest, opts ...grpc.CallOption) (*NodeExpandVolumeResponse, error)
```

Comment:
Just for future consideration: this was a very large PR already so having all the `NodeExpandVolume` plumbing in this PR but not wiring it up makes the PR that much larger for review.

Reply:
Ah yeah, these came with the original branch I picked up, but I can see how they'd be distracting here without a full implementation.  I'm happy to strip them out of here if you'd like?

Reply:
It's fine, just something to keep in mind for next time you've got a big feature you're trying to land is all.